control-plane/agents/cluster/src/server.rs

```diff
@@ -0,0 +1,51 @@
+use anyhow::anyhow;
+use common_lib::transport_api::{ReplyError, ResourceKind};
+use grpc::operations::ha_node::{
+    server::ClusterAgentServer,
+    traits::{ClusterAgentOperations, NodeInfo},
+};
+use std::{net::SocketAddr, sync::Arc};
+use tonic::transport::Server;
+
+pub struct ClusterAgent {
```

Comment:
These don't need to be pub?

Reply:
no, it is being used in control-plane/agents/cluster/src/main.rs

Reply:
no. It is used in [main.rs](https://github.com/openebs/mayastor-control-plane/blob/abad24343a59980563fb8efb751a8c77aa4f3dfa/control-plane/agents/cluster/src/main.rs#L26)

Reply:
They still don't need to be pub, they can be pub(crate).